frontend/package.json

```diff
@@ -5,22 +5,23 @@
   "dependencies": {
     "@emotion/react": "^11.13.5",
     "@emotion/styled": "^11.13.5",
+    "@lordicon/react": "^1.10.0",
```

Comment:
is there a reason why the order of some of the dependencies have been switched even though the version number is the same as the previous? Let's keep these consistent to avoid merge conflicts